mw-webapp/src/logic/waysTable/OwnWaysTable.tsx

```diff
@@ -0,0 +1,50 @@
+import {useEffect, useState} from "react";
+import {WayPreviewDAL} from "src/dataAccessLogic/WayPreviewDAL";
+import {columns} from "src/logic/waysTable/columns";
+import {WaysTable} from "src/logic/waysTable/WaysTable";
+import {WayPreview} from "src/model/businessModelPreview/WayPreview";
+
+/**
+ * Own ways table props
+ */
+interface OwnWaysTableProps {
+
+  /**
+   * User Uuid
+   */
+  uuid: string;
+}
+
+/**
+ * Render table of own ways preview
+ */
+export const OwnWaysTable = (props: OwnWaysTableProps) => {
+  const [ownWays, setOwnWays] = useState<WayPreview[]>([]);
+
+  /**
+   * Load User own ways
+   */
+  const loadOwnWays = async () => {
+    const data = await WayPreviewDAL.getOwnWaysPreview(props.uuid);
+    setOwnWays(data);
+  };
+
+  useEffect(() => {
+    loadOwnWays();
+  }, []);
+
+  const columnsToExclude = ["Owner's name", "Owner's email"];
```

Comment:
magic values could lead to veird regressions; use constants for both files (if you dont see the problem here - call me I will describe)

Reply:
Done